File: 240603/살 수 있는 물건/things-able-to-buy.js
const fs = require("fs");
let input = fs.readFileSync(0).toString();

let money = Number(input);

if (money >= 3000) {
    console.log("book");
} else if (money >= 1000) {
    console.log("mask");
} else {
    console.log("no");
}

File: 251026/살 수 있는 물건/things-able-to-buy.py
n = int(input())

# 가장 비싼 물건 사기
# 만약에 n 1500이면 mask 사야함
if n >= 1000 and n < 3000:
    print("mask")
elif n >= 3000:
    print("book")
else:
    print("no")